community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -186,44 +184,19 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     sampleAllIndicesAndWait()
 
     profileQuery(title = "Node index range seek",
-                 text = """
-                          |Finds nodes using an index seek where the value of the property matches a given prefix string.
-                          |The following query will return all location nodes which have a name property starting with the text 'Lon'.
-                        """.stripMargin,
+                 text = """Finds nodes using an index seek where the value of the property matches a given prefix string. This operator can be used for +LIKE+ and comparators such as <, >, <= and >=""",
```

Comment:
I'd put every comparator inside backticks. BTW backticks are preferred over plus signs; AsciiDoc is changing it's syntax in this area.